base/test/whisper_asr_tests.cpp

```diff
@@ -0,0 +1,58 @@
+#include "stdafx.h"
+#include <boost/test/unit_test.hpp>
+#include "FrameMetadata.h"
+#include "FrameMetadataFactory.h"
+#include "Frame.h"
+#include "Logger.h"
+#include "test_utils.h"
+#include "PipeLine.h"
+#include "FileWriterModule.h"
+#include "FileReaderModule.h"
+#include "FileWriterModule.h"
+#include "WhisperStreamTransform.h"
+#include "Module.h"
+#include<iostream>
+#include<fstream>
+#include<vector>
+
+BOOST_AUTO_TEST_SUITE(whisper_asr_test)
+
+BOOST_AUTO_TEST_CASE(test_asr, *boost::unit_test::disabled())
```

Comment:
any reason this test should be disabled ? It should be possible to run it during builds right ?
the recording test was disabled because all build machines may not have a microphone

Reply:
Nope, this can be enabled. Overlooked while copying tests from sound_record_tests.cpp